website/blog/2022-02-22-towards-better-releases.md

```diff
@@ -0,0 +1,60 @@
+---
+author: Vadim Chelyshov
+title: Towards better releases
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+Hi, Metals folks!
+
+You may have noticed that the previous `0.11.0` release didn't go smoothly.
+Unfortunately, a group of issues was found only after the release was published.
+Some of them were quite critical. The only option to continue to work was to downgrade Metals until `0.11.1` arrived.
+
+We apologize for that!
+
+However, this post is not only about saying sorry.
+The reasons why it's happened weren't a complete surprise for us.
+At the release moment, we were aware of the weaknesses that we had, and they struck us badly.
+
+We are glad to say that there is couple of things that were done to improve Metals release process.
+We hope that with them and your help the chances of repeating the same situation will be minimized.
```

Comment:
```suggestion
```
I don't think the sentence is neccessary if we add the above.

Reply:
I wanted to somehow emphasize in the header that people can help with testing

Reply:
I would just write it explicitely then.

Reply:
ok, just removed it.